libraries/helpers.rb

```diff
@@ -310,6 +310,14 @@ def are_queues_updated?
   config["Scheduling"] != previous_config["Scheduling"] or is_compute_node_bootstrap_timeout_updated?(previous_config, config)
 end
 
+# Verify if CustomSlurmSettings has been updated in the config
+def are_custom_slurm_settings_updated?
```

Comment:
Since this function is monitoring only the `bulk` custom settings, let's be explicit about this and rename the function `are_bulk_custom_slurm_settings_updated?`